.circleci/config.yml

```diff
@@ -271,7 +271,7 @@ jobs:
       - run:
           name: Check for known security issues in dependencies
           # TODO remove ignore when https://github.com/filecoin-project/rust-fil-proofs/issues/1403 resolved
-          command: cargo audit --ignore RUSTSEC-2021-0011
+          command: cargo audit --ignore RUSTSEC-2021-0011 --ignore RUSTSEC-2021-0070
```

Comment:
Can you leave a comment like the one on 273?
Also, be sure to label that 273 is for `RUSTSEC-2021-0011`, while the new comment is for `RUSTSEC-2021-0070`, so we know which is which.